core/src/main/java/io/parsingdata/metal/data/ParseState.java

```diff
@@ -95,7 +101,8 @@ public ParseState withSource(final Source source) {
     @Override
     public String toString() {
         final String iterationString = iterations.isEmpty() ? "" : ";iterations:" + iterations.toString();
-        return getClass().getSimpleName() + "(source:" + source + ";offset:" + offset + ";order:" + order + iterationString + ")";
+        final String targetsString = references.isEmpty() ? "" : ";references:" + references.toString();
```

Comment:
I guess that the 'references' concept was renamed somewhere along the line? I would expect `referencesString` here.